Bootcamp dev

FREDVUNI opened the pull request:
# Ticket No
- https://trello.com/c/lW3WXqYC
- https://trello.com/c/94zi5wpb
![Screenshot (112)](https://user-images.githubusercontent.com/41730664/185973731-35487c27-fb3f-45cf-9151-263a3d6f81f3.png)

# Summary of changes
- Changed the navigation menu title from **angie** to **Project Zoe** and also menu item **Chat** to **Connect**

# How should this be tested? [Screenshots, Video or Type instructions]
- screenshot

# Notes for reviewers
- Notes for reviewers here

# Out of scope
-  Out of scope here

jacksono commented on src/modules/login/Login.tsx:
```diff
@@ -32,10 +36,12 @@ function Login() {
         history.push(localRoutes.profile);
       },
       () => {
+        console.log("whyy")
```
This console log should be removed.

jacksono commented on src/modules/login/Login.tsx:
```diff
@@ -23,7 +23,11 @@ function Login() {
   const history = useHistory();
   const onSubmit = (data: any, actions: FormikHelpers<any>) => {
-    post(
+    // dispatch(handleLogin({}));
+    Toast.success('Authentication success');
+    history.push(localRoutes.profile);
+    if ((1 + 3) == 2) {
+      post(
```
There should be no changes here

jacksono commented on src/index.tsx:
```diff
@@ -1,37 +1,37 @@
-    return event;
-  },
-});
+// const SENTRY_STATUS = process.env.NODE_ENV === 'production';
+// Sentry.init({
+//   dsn: process.env.REACT_APP_SENTRY_DSN,
+//   autoSessionTracking: SENTRY_STATUS,
+//   integrations: [
+//     new Integrations.BrowserTracing(),
+//   ],
+//   tracesSampleRate: 0.5,
+//   beforeSend: (event) => {
+//     if (
+//     // Exclude "localhost" envs from issue tracker
+//       window.location.hostname === 'localhost'
+//     ) {
+//       return null;
+//     }
+//     return event;
+//   },
+// });
```
This should not be commented out

jacksono commented on src/index.tsx:
```diff
@@ -1,37 +1,37 @@
 import 'react-toastify/dist/ReactToastify.css';
-import * as Sentry from '@sentry/react';
-import { Integrations } from '@sentry/tracing';
+// import * as Sentry from '@sentry/react';
+// import { Integrations } from '@sentry/tracing';
```
This should not be commented out

jacksono commented on src/components/layout/NavMenu.tsx:
```diff
@@ -195,7 +197,8 @@ const NavMenu = (props: any) => {
         justify="center"
       >
-        <img src={appLogo} alt="logo" className={classes.logo} />
+        {/* <img src={appLogo} alt="logo" className={classes.logo} /> */}
```
It should be safe to delete this line since we are replacing it with the line below it.

jacksono commented on src/components/layout/NavMenu.tsx:
```diff
@@ -56,7 +56,7 @@ const routes: IAppRoute[] = [
   },
   {
-    name: 'Chat',
+    name: 'Connect',
```
👍🏾

jacksono commented on package.json:
```diff
@@ -18,6 +18,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.54",
     "@material-ui/pickers": "^3.2.10",
+    "@material-ui/styles": "^4.11.5",
```
Why did you need to add this package?

FREDVUNI replied:
There was an error raised by material-ui and I found out that installing styles would solve the issue. I'm going to uninstall it though because we have since made quite a number of changes

jacksono commented on .eslintignore:
```diff
@@ -4,3 +4,4 @@ my-app*
 packages/react-scripts/template
 packages/react-scripts/fixtures
 fixtures/
+src/*
```
This folder should not be ignored

jacksono reviewed (changes requested):
@FREDVUNI  I have left a few comments that need to be addressed.

FREDVUNI commented:
Alright. This is probably why my client side is currently not working as it
should

On Tue, Aug 23, 2022, 11:58 AM Jackson Onyango ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> @FREDVUNI <https://github.com/FREDVUNI> I have left a few comments that
> need to be addressed.
> ------------------------------
>
> In src/modules/login/Login.tsx
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952339304>
> :
>
> > @@ -32,10 +36,12 @@ function Login() {
>
>          history.push(localRoutes.profile);
>
>        },
>
>        () => {
>
> +        console.log("whyy")
>
>
> This console log should be removed.
> ------------------------------
>
> In src/modules/login/Login.tsx
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952339705>
> :
>
> > +    // dispatch(handleLogin({}));
>
> +    Toast.success('Authentication success');
>
> +    history.push(localRoutes.profile);
>
> +    if ((1 + 3) == 2) {
>
> +      post(
>
>
> There should be no changes here
> ------------------------------
>
> In src/index.tsx
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952340494>
> :
>
> > +// const SENTRY_STATUS = process.env.NODE_ENV === 'production';
>
> +// Sentry.init({
>
> +//   dsn: process.env.REACT_APP_SENTRY_DSN,
>
> +//   autoSessionTracking: SENTRY_STATUS,
>
> +//   integrations: [
>
> +//     new Integrations.BrowserTracing(),
>
> +//   ],
>
> +//   tracesSampleRate: 0.5,
>
> +//   beforeSend: (event) => {
>
> +//     if (
>
> +//     // Exclude "localhost" envs from issue tracker
>
> +//       window.location.hostname === 'localhost'
>
> +//     ) {
>
> +//       return null;
>
> +//     }
>
> +//     return event;
>
> +//   },
>
> +// });
>
>
> This should not be commented out
> ------------------------------
>
> In src/index.tsx
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952340900>
> :
>
> > +// import * as Sentry from ***@***.***/react';
>
> +// import { Integrations } from ***@***.***/tracing';
>
>
> This should not be commented out
> ------------------------------
>
> In src/components/layout/NavMenu.tsx
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952341675>
> :
>
> > @@ -195,7 +197,8 @@ const NavMenu = (props: any) => {
>
>          alignContent="center"
>
>          justify="center"
>
>        >
>
> -        <img src={appLogo} alt="logo" className={classes.logo} />
>
> +        {/* <img src={appLogo} alt="logo" className={classes.logo} /> */}
>
>
> It should be safe to delete this line since we are replacing it with the
> line below it.
> ------------------------------
>
> In src/components/layout/NavMenu.tsx
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952341895>
> :
>
> > @@ -56,7 +56,7 @@ const routes: IAppRoute[] = [
>
>      icon: TodayIcon,
>
>    },
>
>    {
>
> -    name: 'Chat',
>
> +    name: 'Connect',
>
>
> 👍🏾
> ------------------------------
>
> In package.json
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952342717>
> :
>
> > @@ -18,6 +18,7 @@
>
>      ***@***.***/icons": "^4.9.1",
>
>      ***@***.***/lab": "^4.0.0-alpha.54",
>
>      ***@***.***/pickers": "^3.2.10",
>
> +    ***@***.***/styles": "^4.11.5",
>
>
> Why did you need to add this package?
> ------------------------------
>
> In .eslintignore
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#discussion_r952343125>
> :
>
> > @@ -4,3 +4,4 @@ my-app*
>
>  packages/react-scripts/template
>
>  packages/react-scripts/fixtures
>
>  fixtures/
>
> +src/*
>
>
> This folder should not be ignored
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kanzucodefoundation/project-zoe-client/pull/118#pullrequestreview-1081738271>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ6ME2CMXJVLLVI7PE22HQ3V2SHDRANCNFSM57ILLABA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

FREDVUNI committed:
Changed navigation menu title and item

jacksono reviewed (approved):
LGTM

FREDVUNI committed:
Added an edit button icon and a pop up modal dialog box

FREDVUNI commented:
# Ticket No
- https://trello.com/c/00zJRFve

![Screenshot (118)](https://user-images.githubusercontent.com/41730664/190146234-a1732c1f-33d1-481f-a7a5-3929e4b3e6da.png)

![Screenshot (114)](https://user-images.githubusercontent.com/41730664/190145410-7b8ba685-da41-48ec-a8bb-4c07b8b1031e.png)

![Screenshot (116)](https://user-images.githubusercontent.com/41730664/190145417-bf511112-853b-4987-aa7c-75d5dc492ffd.png)

# Summary of changes
- Edit person details through pop-up modal in place within the Persons Table
- Closing modal dialog returns to the people page when the URL has parameter showEdit

# How should this be tested? [Screenshots, Video or Type instructions]
- screenshot


# Notes for reviewers
- Notes for reviewers here


# Out of scope
-  Out of scope here

FREDVUNI committed:
Moved the send email button and removed coming soon text

FREDVUNI commented:
# Ticket No
- https://trello.com/c/DCDA6rAH

![Screenshot (120)](https://user-images.githubusercontent.com/41730664/190843640-7af78ae1-7663-4216-ac76-19fa91e27f04.png)

# Summary of changes
- Removed coming soon text and moved the send email button

# How should this be tested? [Screenshots, Video or Type instructions]
- screenshot


# Notes for reviewers
- Notes for reviewers here


# Out of scope
-  Out of scope here

FREDVUNI committed:
Changed landing page from profile to dashboard

FREDVUNI commented:
# Ticket No
- https://trello.com/c/tUAwrgWp

# Summary of changes
- Changed the landing page from profile to dashboard

# How should this be tested? 
- Demo


# Notes for reviewers
- Notes for reviewers here


# Out of scope
-  Out of scope here

Roland-Sankara commented:
LGTM
